### PR TITLE
Fix #1935 Converted constants to Kotlin

### LINKED ISSUE #1935: Converting Constants to Kotlin and use it consistently in whole project
**Is your feature request related to a problem? Please describe.**

Constants are still in java. It can be converted into Kotlin. There must be a consistent way to use Constants in whole project.

**Describe the solution you'd like**

Convert it to Kotlin. Whenever a constant is required, import it from its name.

**Additional context**

Parent Issue: #1826

### PR DESCRIPTION
**Pull request by akkywadhwa**
Fixes #1935

**Screenshots**

### CODE REVIEW

File: core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.utils
+
+object Constants {
```
Comment (abdulwd) [inline core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt]: We don't need an object to store constants. You can declare them directly in the file.

Comment (akkywadhwa) [reply core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt]: @abdulwd If I store constants without any class/object, I would need to it import it as
`import org.kiwix.kiwixmobile.core.utils.VARIABLE_NAME` There would be no use of using "Constants" as the filename. I thought using `object` will make the code more clear.

If using `org.kiwix.kiwixmobile.core.utils.VARIABLE_NAME` seems okay, I can make the changes.

Comment (abdulwd) [reply core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt]: I rarely check how's the variable imported. I use code navigation `Command + Click` to jump to the definition.

Comment (akkywadhwa) [reply core/src/main/java/org/kiwix/kiwixmobile/core/utils/Constants.kt]: @abdulwd I have made the required changes, please check.

**Pull request merged**